protocol/messenger_communities.go

```diff
@@ -1106,6 +1137,220 @@ func (m *Messenger) handleSyncCommunity(messageState *ReceivedMessageState, sync
 	return nil
 }
 
+func (m *Messenger) InitHistoryArchiveTasks(communities []*communities.Community) {
+
+	dispatchMagnetlink := func(communityID types.HexBytes) {
+		err := m.dispatchMagnetlinkMessage(communityID)
+		if err != nil {
+			m.logger.Debug("failed to dispatch magnetlink message", zap.Error(err))
+		}
+	}
+
+	for _, c := range communities {
+
+		if c.Joined() {
+			settings, err := m.communitiesManager.GetCommunitySettingsByID(c.ID())
+			if err != nil {
+				log.Println("failed to get community settings", err)
+				m.logger.Debug("failed to get community settings", zap.Error(err))
```

Comment:
I've doubled the logs here so I can see something in stdout. Happy to remove them for final version, but it's very helpful to see what's going on in realtime..